### ISSUE TITLE
fallbackLng does not allow for different translations

### ISSUE THREAD

**Issue by Comum** [i18next/i18next#1577]
## 🐛 Bug Report

Since the change of the fallbackLng we are getting an error when fetching any translations.

## To Reproduce

We have a multipage app, that allows for different langs. Every time the user changes a language the pages reloads with the new values. At all times we only load one language object (ie. en, ja, ...).

We think our issue comes from the the fallback not being present in our resources object.

```js
const { lng = fallbackLng } = options;

// fallbackLng = 'en'
// lng = 'ja'

  i18n.use(initReactI18next).init({
    lng,
    whitelist,
    defaultNS,
    interpolation,
    fallbackLng,
    parseMissingKeyHandler,
    react: {
      wait: true,
    },
    resources: {
      [lng]: {
        [defaultNS]: getTranslates(lng),
      },
    },
    ...options,
  });
```

so basically our resources will be something like:

```
    resources: {
        'ja': { ... }
    }
```

and we think that since it does not have the fallback value of 'en', it somehow impacts the returned translation.

## Expected behavior

Actually seeing the translations.

## Your Environment

- *runtime version*: node v12.16.1
- *i18next version*: 19.8.9
- *os*: Mac, Windows

**Comment by Comum** [comment]
Works with version 19.9.2
